website/content/docs/connect/cluster-peering/k8s.mdx

```diff
@@ -25,9 +25,26 @@ You must implement the following requirements to create and use cluster peering
 - At least two Kubernetes clusters
 - The installation must be running on Consul on Kubernetes version 0.47.1 or later
 
+### Pre-setup 
+
+1. After provisioning a Kubernetes cluster and setting up your kubeconfig file to manage access to multiple Kubernetes clusters, export the Kubernetes context names for future with `kubectl`. 
+
+  You can discover the context names using either `kubectl config current-context` for the current cluster you are on, or by issuing `kubectl config get-contexts` to get all configured contexts for your kubeconfig file.  
+
+  ```shell-session
+  $ export CLUSTER1_CONTEXT=<CONTEXT for first Kubernetes cluster>
+  $ export CLUSTER2_CONTEXT=<CONTEXT for second Kubernetes cluster>
+  ```
+
 ### Helm chart configuration
 
-To establish cluster peering through Kubernetes, deploy clusters with the following Helm values.
+1. To establish cluster peering through Kubernetes, create a values.yaml file with the following Helm values. 
+   
+  With these values,
+  the servers in each cluster will be exposed over a Kubernetes Load balancer service. This service can be customized
+  using [`server.exposeService`](/docs/k8s/helm#v-server-exposeservice). 
```

Comment:
```suggestion
  These values expose the servers in each cluster will over a Kubernetes Load balancer service. You can configure the [`server.exposeService`](/docs/k8s/helm#v-server-exposeservice) value to customize the service.
```
The more I think about, the more I think that we should position the content under "Pre-setup" and "Helm chart configuration" as the first step in the procedure with a heading like "Prepare your Kubernetes environment". We would just need to indicate that the practitioner only needs to perform this part of the procedure once (right?).

Reply:
I've added preparation for install as I believe that is the better way to describe the retrieval of kube contexts and creation of the values file under one heading.